deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -338,6 +337,8 @@ def wait_to_remoted_key_update(wazuh_log_monitor):
     Raises:
         TimeoutError: if could not find the remoted key loading log.
     """
+    file.truncate_file(LOG_FILE_PATH)
```

Comment:
Can you add a comment here explaining why it is important to truncate the log before waiting for the reload of the client.keys?

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1101/commits/fd00b88abcd5a4dcf3af786dc0a7f86fb3d3a662